conans/test/unittests/tools/files/test_tool_copy.py

```diff
@@ -169,6 +170,29 @@ def test_excludes(self):
         copy(None, "*.txt", folder1, folder2, excludes=("*Test*.txt", "*Impl*"))
         self.assertEqual(['MyLib.txt'], os.listdir(folder2))
 
+    def test_excludes_hidden_files(self):
+        folder1 = temp_folder()
+        folder_foo = os.path.join(folder1, "foo")
+        folder_foo_bar = os.path.join(folder_foo, "bar")
+        folder_hidden = os.path.join(folder1, ".hiddenfolder")
+        mkdir(folder_foo)
+        mkdir(folder_foo_bar)
+        mkdir(folder_hidden)
+
+        save(os.path.join(folder1, "file1.txt"), "")
+        save(os.path.join(folder1, ".hiddenfile"), "")
+        save(os.path.join(folder_foo, "file2.txt"), "")
+        save(os.path.join(folder_foo, ".hiddenfile2"), "")
+        save(os.path.join(folder_hidden, "file3.txt"), "")
+        save(os.path.join(folder_foo_bar, "file4.txt"), "")
```

Comment:
You might do:
```python
save_files(folder1, {"foo/bar/.hiddenfolder/filer.txt": "",
                      "foo/bar/...
```
That might make a bit more evident to the test reader all the files and folders involved.

Reply:
Ohh, didn't know that!